design/API.md

```diff
@@ -328,133 +343,768 @@ Get the status of autotune.
+                            "max": 2107.212121,
+                            "mean": 31.91
                         }
-                    }
+                    },
+                    "name": "request_count"
                 }
-            },
+            ],
+            "deployment_name": "tfb-qrh-sample",
+            "namespace": "default",
+            "containers": [{
+                "image_name": "kruize/tfb-qrh:1.13.2.F_et17",
+                "container_name": "tfb-server",
+                "container_metrics": [
+                    {
+                        "datasource": "prometheus",
+                        "summary_results": {
+                            "percentile_info": {
+                                "99p": 82.59,
+                                "97p": 64.75,
+                                "95p": 8.94,
+                                "50p": 0.63,
+                                "99.9p": 93.48,
+                                "100p": 30000,
+                                "99.99p": 111.5,
+                                "99.999p": 198.52
+                            },
+                            "general_info": {
+                                "min": 2.15,
+                                "max": 2107.212121,
+                                "mean": 31.91
+                            }
+                        },
+                        "name": "memoryRequest"
+                    },
+                    {
+                        "datasource": "prometheus",
+                        "summary_results": {
+                            "percentile_info": {
+                                "99p": 82.59,
+                                "97p": 64.75,
+                                "95p": 8.94,
+                                "50p": 0.63,
+                                "99.9p": 93.48,
+                                "100p": 30000,
+                                "99.99p": 111.5,
+                                "99.999p": 198.52
+                            },
+                            "general_info": {
+                                "min": 2.15,
+                                "max": 2107.212121,
+                                "mean": 31.91
+                            }
+                        },
+                        "name": "cpuRequest"
+                    }
+                ]
+            }],
             "type": "training"
-        }
+        }],
+        "experiment_id": "04c99daec35563782c29f17ebe568ea96065a7b20b93eb47c225a2f2ad769445",
+        "deployment_name": "tfb-qrh-sample",
+        "info": {"trial_info": {
+            "trial_id": "",
+            "trial_num": 1,
+            "trial_result_url": "http://localhost:8080/listExperiments?experiment_name=quarkus-resteasy-autotune-min-http-response-time-db"
+        }}
     },
+}
+```
+
+`GET /listTrialStatus?experiment_name=<experiment name>&trial_num=<trial number>` Gives the Trial status of the
```

Comment:
I think we should use trial_number instead of trial_num to keep it consistent with the parameter names used in HPO API